.github/workflows/test-publish.yml

```diff
@@ -0,0 +1,25 @@
+name: Test publish
+
+on:
+  workflow_call:
+
+jobs:
+  test-publish:
+    runs-on: ubuntu-20.04
```

Comment:
Why not ubuntu latest ?

Reply:
the playwright tests are not working on latest

Reply:
Is there a git issue reference that would explain it?

Reply:
I think it's just because we use an old version of playwright.
Ubuntu 22.04 should be supported starting from 1.24